DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1364,7 +1364,7 @@ public partial class DMParser : Parser<Token> {
                     DMASTExpression? expression = Expression();
                     if (expression == null) {
                         if (expressions.Count == 0) {
-                            Error("Expected an expression");
+                            Error(WarningCode.BadExpression, "Expected an expression");
                         } else {
                             //Eat a trailing comma if there's at least 1 expression
                             break;
```

Comment:
This needs to break out of the loop, now that the `Error()` call isn't throwing.
```suggestion
                            Error(WarningCode.BadExpression, "Expected an expression");
                        }
                        
                        //Eat a trailing comma
                        break;
```